charts/caraml/Chart.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v2
+name: caraml
+description: A Helm chart for Kubernetes
```

Comment:
I'd rephrase it to something referencing CaraML

Reply:
Updated the description